packages/jest-worker/src/__tests__/worker.test.js

```diff
@@ -41,14 +43,19 @@ beforeEach(() => {
 
 afterEach(() => {
   jest.resetModules();
+  // eslint-disable-next-line no-native-reassign
+  process = properProcess;
 });
 
 it('passes fork options down to child_process.fork, adding the defaults', () => {
   const child = require.resolve('../child');
+
+  Object.assign(process, {execArgv: ['--inspect', '-p']});
```

Comment:
The `process` => `properProcess` dance does not prevent from modifying the global object, since `Object.assign` will assign to the real `process`. Instead you might want to save `execArgv` at the beginning, and re-assign to `process` on the `afterEach`.

Reply:
I don't really care that I modify it, it's sandboxed for this test in particular, right?

Good idea on just messing with the part I need to mess with, though